src/index.js

```diff
@@ -195,6 +195,9 @@ export default class PerfectScrollbar {
   }
 
   onScroll(e) {
+    if (element.classList.contains('ps-disabled')) {
```

Comment:
Have you actually tested this? i may merged this too prematurely, since i think here you meant to use `this.element` (and in other similar places?)

Reply:
The other were fine, sorted this one with [bd2e703](https://github.com/meganz/perfect-scrollbar/commit/bd2e7037e8961575e0f4f1f255938355022e37dd)